runelite-client/src/main/java/net/runelite/client/plugins/friendschat/FriendsChatPlugin.java

```diff
@@ -715,4 +734,39 @@ private void colorIgnoredPlayers(Color ignoreColor)
 			listWidget.setTextColor(ignoreColor.getRGB());
 		}
 	}
+
+	@Subscribe
+	public void onWidgetMenuOptionClicked(WidgetMenuOptionClicked event)
+	{
+		if ((event.getWidget() == WidgetInfo.FIXED_VIEWPORT_FRIENDS_CHAT_TAB
+			|| event.getWidget() == WidgetInfo.RESIZABLE_VIEWPORT_FRIENDS_CHAT_TAB
+			|| event.getWidget() == WidgetInfo.RESIZABLE_VIEWPORT_BOTTOM_LINE_FRIEND_CHAT_TAB)
+			&& CLEAR_RECENT_OPTION.equals(event.getMenuOption())
+			&& CLEAR_RECENT_TARGET.equals(Text.removeTags(event.getMenuTarget())))
```

Comment:
It doesn't look like tags are added to the menu target by MenuManager, why is `removeTags()` necessary?

Reply:
I was copying `Menu Entry Swapper`'s shift click stuff. Though that the colour that the target has was added to the text, but I'll remove it.

Reply:
So it turns out that `removeTags` is there for a good reason
![image](https://user-images.githubusercontent.com/2979691/109432302-6b151880-7a02-11eb-8ce4-6054ecf9e6fd.png)